apps/src/music/redux/musicRedux.ts

```diff
@@ -25,6 +27,26 @@ const musicSlice = createSlice({
     },
     setCurrentPlayheadPosition: (state, action: PayloadAction<number>) => {
       state.currentPlayheadPosition = action.payload;
```

Comment:
Is it normal to call it `payload`?  It feels like it might be more readable if we called it `action.playheadPosition` or similar?

Reply:
Is there any chance payload could be an object and we could have a named value inside that?  Not sure what is the idiomatic approach though...

Reply:
Great callout though! The whole 'payload' naming reminds me a bit of [Flux](https://github.com/facebookarchive/flux) which I had used a bit before Redux became more mainstream